content/config/daemon/proxy.md

```diff
@@ -1,44 +1,28 @@
 ---
-description: Learn about controlling and configuring the Docker daemon using systemd
-keywords: dockerd, daemon, systemd, configuration, proxy, networking
-title: Configure the daemon with systemd
+description: Learn how to configure the Docker daemon to use an HTTP proxy
+keywords: dockerd, daemon, configuration, proxy, networking, http_proxy, https_proxy, no_proxy, systemd, environment variables
+title: Configure the daemon to use a proxy
 aliases:
   - /articles/host_integration/
   - /articles/systemd/
   - /engine/admin/systemd/
   - /engine/articles/systemd/
 ---
 
-This page describes how to customize daemon settings when using systemd.
+<a name="httphttps-proxy"><!-- included for deep-links to old section --></a>
 
-## Custom Docker daemon options
+You can configure HTTP proxy settings for the daemon in two ways:
```

Comment:
I think we may need a short introduction on this page; we jump straight in "You can configure..", but proxy settings have been confusing for many users (most notably because there's proxy settings for the `daemon`, `containers`, `build` and the CLI); perhaps we should add a short intro to describe what this specific proxy is used for?

In this case, it's for the daemon itself, so networking performed by the daemon, which roughly is for i

- Interactions with registries (authentication through the API, pushing, pulling, and searching images)
- When running a swarm; networking and communication with other nodes in the swarm (?)

Reply:
Good call, adding something here.